core-bundle/config/listener.yaml

```
@@ -605,6 +605,11 @@ services:
             - '@router'
             - '@translator'
 
+    contao.messenger.message_listener:
+        class: Contao\CoreBundle\Messenger\EventListener\MessageListener
+        arguments:
+            - '@logger'
```

Instead of the general logger we could also inject the Contao system logger. This way you don't have to create the `ContaoContext` yourself.
```suggestion
            - '@monolog.logger.contao.error'
```